lib/buckets.ps1

```diff
@@ -103,6 +103,24 @@ function Find-Manifest($app, $bucket) {
     return $app, $manifest, $bucket, $url
 }
 
+function Convert-RepositoryUri {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory, Position = 0, ValueFromPipeline = $true)]
+        [String] $Uri
+    )
+
+    process {
```

Comment:
We can omit the `process` block for such a small function

Reply:
Enabling pipeline needs explicit `process`, and in fact, if not specified, the script block is put under `begin`.

Reply:
Oh, didn't know that 😅